server/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -23,6 +23,18 @@
 -->
 
 <urlrewrite>
+  <rule>
```

Comment:
If not removing the old API, you should put a toggle here. See the class SparkOrRails or something.

Reply:
Removed rails based API.